Fix conditions for completion of demo game

kjetil-kilhavn opened the pull request:
The demo game states that you should deliver the RFC to the developers. Added condition that you have to be in the same room as Bill the developer.

Seems some space characters were removed too - that wasn't intentional.

kjetil-kilhavn committed:
Fix conditions for completion of demo game

The demo game states that you should deliver the RFC to the developers.
Added condition that you have to be in the same room as bill the developer.

AndreaBorgia-Abo commented:
> Seems some space characters were removed too - that wasn't intentional.

You could remove the whole space change-block directly from GH, it is quicker than via the GUI.

kjetil-kilhavn committed:
Add trailing spaces back to statements which weren't changed

Trailing spaces added back to statements which weren't changed to reduce the "foot print" of the pull request.

kjetil-kilhavn commented:
Trailing spaces inserted again, so now there's one more error message from abaplint :-)
However, the "footprint" of the pull request is reduced to the statements it was necessary to change for the correction.

Ennowulff commented:
thanks!
